docs/getting-started/roles/sqlalchemy/basic_roles.rst

```diff
@@ -0,0 +1,270 @@
+=====================
+Roles with SQLAlchemy
+=====================
+
+The :py:data:`sqlalchemy_oso.roles` module provides out-of-the-box Role-Based Access Control features that
+let you create a roles system  with a few lines of code, and specify role permissions
+in a declarative oso policy.
+
+This guide walks you through how to use ``sqlalchemy_oso`` to add basic roles to a multi-tenant app.
+
+    Note: we're using a Flask app for this example, but the
+    ``sqlalchemy_oso`` library can be used with any Python application.
+
+1. Set up the application
+=========================
+
+Install the oso SQLAlchemy package
+----------------------------------
+
+Install the ``sqlalchemy_oso`` package and import it in your code.
+Alternatively, if you are starting from scratch and need a
```

Comment:
The "if you need a requirements.txt" file part of this doesn't really make sense to me. I'd just drop that part.

Reply:
I think this was from my outline, where I was cribbing prose from the stripe docs.

Reply:
fixed.